pipeline/extract.py

```diff
@@ -0,0 +1,200 @@
+'''File used for extracting information from bandcamp's website and API'''
+
+
+import asyncio
+import json
+import logging
+from time import sleep
+
+import aiohttp
+import requests as req
+from bs4 import BeautifulSoup
```

Comment:
Alphabetical order here to improve readability by a tiny amount
```suggestion
import aiohttp
from bs4 import BeautifulSoup
import requests as req
```